### ISSUE TITLE
v1.19 buffer by default?

### ISSUE THREAD

**Issue by nr1995** [Genymobile/scrcpy#2626]
Sorry, not a feature request, but rather just a question.
Is the buffer introduced in v1.19 enabled by default?
Thanks.

**Comment by rom1v** [comment]
No, you have to explicitly pass `--display-buffer=XXX`, where XXX is the buffer duration in milliseconds.

**Comment by nr1995** [comment]
As not a programmer guy myself,
when you say “pass”, do you mean create a shortcut like this?:
"D:\scrcpy\scrcpy-noconsole.exe" --display-buffer=XXX

**Comment by rom1v** [comment]
https://github.com/Genymobile/scrcpy/blob/master/FAQ.md#command-line-on-windows

**Comment by nr1995** [comment]
Sorry just one more thing,
when I'm editing my copy of _scrcpy-noconsole.vbs_
where do I add the arguments?

strCommand = "cmd /c scrcpy.exe"

For Each Arg In WScript.Arguments
    strCommand = strCommand & " """ & replace(Arg, """", """""""""") & """"
Next

CreateObject("Wscript.Shell").Run strCommand, 0, false
